ftw/solr/locales/update.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
there is already an `bin/i18n-build` provided by development buildout. i think this script is not necessary.
also, there are longer any translations, so the locales directory could be removed?

Reply:
removed